ONSdigital/eq-survey-runner
# Refine cloudwatch logging

**_What**_

Removes default apache monitoring rules and associated alerts.
Moves EQ_SR_LOG_LEVEL and EQ_LOG_GROUP into settings.py
Defines default values for both EQ_SR_LOG_LEVEL and EQ_LOG_GROUP to ease local development

**_How to test**_

1) Checkout eq-terraform and modify survey-rubber.tf to use this branch
2) run terraform apply
3) Upon sucessful smoke test, check in CloudWatch to verify that logs are still being collected, but that the apache alerts are no longer being created

**_Who can test**_

Anyone except @weapdiv-david
